com.unity.multiplayer.mlapi/Editor/PostProcessScene.cs

```diff
@@ -11,13 +11,7 @@ public class NetworkScenePostProcess : MonoBehaviour
         [PostProcessScene(int.MaxValue)]
         public static void ProcessScene()
         {
-            //If we are in playmode (editor or stand alone) we do not want this to execute
-            if (Application.isPlaying)
```

Comment:
I wonder why the code before skipped out in this case.  Any recollections @TwoTenPvP ?

Reply:
This has been added as a fix by Noel but with the new better fix isn't necessary anymore I think.